docs/notebooks/proton_radiography_particle_tracing.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p>I would not be against actually calling it <code>SimulatedProtonRadiography</code> in the module! It's more explicit that way. @namurphy, @rocco8773, whatcha think ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I generally support longer class names that are clear and unambiguous, so `SimulatedProtonRadiography` sounds good to me.

Reply:
I'm all for more explicit names!  My only issue is using "Simulation" outside the simulation sub-pkg, but I'll get over that.

Reply:
How about `SyntheticProtonRadiograph`? I'd like to eventually include other methods of predicting radiographs from field grids to the same class (for example, there is a method using integration that works for linear small-deflection cases) and those methods are not all "simulations".

Reply:
Sure, Radiograph sounds awesome. And I'm completely fine with it staying out of Simulation.

Reply:
Great, class name updated to "SyntheticProtonRadiograph"